### PR TITLE
[DOC] update remote workflow document when using mlrun version > 1.7.0 

### PR DESCRIPTION
**Pull request by GiladShapira94**
https://iguazio.atlassian.net/browse/ML-7801

### CODE REVIEW

File: docs/concepts/scheduled-jobs.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun > 1.7.1 -  when running remote/schedule workflow, the remote workflow pull/extract the remote source content to the running pod but load the project configuration from MLRun db and not from the project.yaml in the remote source.
```
Comment (jillnogold) [inline docs/concepts/scheduled-jobs.md]: From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the `project.yaml` in the remote source.

Comment (GiladShapira94) [reply docs/concepts/scheduled-jobs.md]: Thanks 👍

File: docs/concepts/scheduled-jobs.md [hunk 2]
```diff
@@ -69,6 +69,8 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml in the remote source.
 ```
```
Comment (yaelgen) [inline docs/concepts/scheduled-jobs.md:81]: I might have add something like:
```suggestion
The remote files are primarily retrieved for the project_setup (if it exists) and for syncing function files. This behavior may be unexpected for users who rely on project.yaml in the remote source for configuration.
Be sure to update any workflows or automation that rely on this file to ensure consistent configuration management.
```
```

**Review by yaelgen — CHANGES_REQUESTED** [review]
LGTM, added a suggestion

File: docs/concepts/scheduled-jobs.md [hunk 3]
```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
...
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+1. The [project_setup](https://docs.mlrun.org/en/stable/projects/project-setup.html) that may affect the project configuration (if it exists).
+2. Syncing function files.
+This behavior may be unexpected for users who rely on project.yaml in the remote source (for the project configuration).
+Be sure to update MLRun DB with the latest project configuration to ensure consistent configuration management (use `project.save()`).
 ```
```
Comment (yaelgen) [inline docs/concepts/scheduled-jobs.md:80]: 🫶

**Review by yaelgen — APPROVED** [review]

File: docs/concepts/scheduled-jobs.md [hunk 4]
```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+1. The [project_setup](https://docs.mlrun.org/en/stable/projects/project-setup.html) that may affect the project configuration (if it exists).
```
Comment (jillnogold) [inline docs/concepts/scheduled-jobs.md]: correct link is:  [project_setup](../projects/project-setup.md)

Comment (GiladShapira94) [reply docs/concepts/scheduled-jobs.md]: Thanks i change the link to the correct one

File: docs/concepts/scheduled-jobs.md [hunk 5]
```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+1. The [project_setup](https://github.com/mlrun/mlrun/projects/project-setup.md) that may affect the project configuration (if it exists).
```
Comment (jillnogold) [inline docs/concepts/scheduled-jobs.md]: If you're not going with Yael's suggestion, pls change the 1. and 2. to "- "

File: docs/concepts/scheduled-jobs.md [hunk 6]
```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+- The [project_setup](https://github.com/mlrun/mlrun/projects/project-setup.md) that may affect the project configuration (if it exists).
```
Comment (jillnogold) [inline docs/concepts/scheduled-jobs.md]: sorry - should be: (.../projects/project-setup.md)

**Review by jillnogold — APPROVED** [review]

File: docs/concepts/scheduled-jobs.md [hunk 7]
```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
...
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+- The [project_setup](../projects/project-setup.md) that may affect the project configuration (if it exists).
+- Syncing function files.
+This behavior may be unexpected for users who rely on project.yaml in the remote source (for the project configuration).
+Be sure to update MLRun DB with the latest project configuration to ensure consistent configuration management (use `project.save()`).
```
Comment (jillnogold) [inline docs/concepts/scheduled-jobs.md:79]: add another row with ``` to resolve the publishing

**Comment by ghost** [comment]
I think we need to emphasize in this explanation that this behavior only affects project configurations and not function configurations (such as function node selectors, request/limit of memory or cpu, etc..).

**Comment by GiladShapira94** [comment]
Will be release in - https://github.com/mlrun/mlrun/pull/6775

**Pull request closed without merging**